server/src/main/kotlin/org/javacs/kt/SourceFiles.kt

```diff
@@ -106,7 +106,13 @@ class SourceFiles(private val sp: SourcePath) {
     private fun readFromDisk(file: Path): SourceVersion? {
         if (!Files.exists(file)) return null
 
-        val content = Files.readAllLines(file).joinToString("\n")
+        var content = ""
+
+        try {
+            content = Files.readAllLines(file).joinToString("\n")
```

Comment:
It will be cleaner if you return from this line instead of saving the file contents to a variable

Reply:
Storing the contents in a variable is fine IMO. Since the string is going to be wrapped into a `SourceVersion` anyway, having two `return`s would entail some code duplication.